restclient/src/main/java/com/box/l10n/mojito/rest/client/RepositoryClient.java

```diff
@@ -224,7 +224,7 @@ public void updateRepository(String name, String newName, String description, Bo
         }
 
         if (createdBefore != null) {
-            filterParams.put("createdBefore", createdBefore.toString());
+            filterParams.put("createdBefore", createdBefore);
```

Comment:
One solution might be to prevent having the `+` by using some other representation like a timestamp: `filterParams.put("createdBefore", String.valueOf(createdBefore.getMillis()))`;

Reply:
That worked nicely and is much simpler that the initial PR (I previously assumed only dates in ISO 8601 were expected on the API side.)